CMakeLists.txt

```diff
@@ -20,3 +29,16 @@ set_property(TARGET oeedger8r PROPERTY POSITION_INDEPENDENT_CODE on)
 enable_testing()
 add_subdirectory(test)
 
+option(CODE_COVERAGE "Enable code coverage testing" OFF)
+if(CODE_COVERAGE)
+  message("Building for code coverage.")
+  target_compile_options(oeedger8r PRIVATE -g -O0 -coverage -fprofile-arcs
```

Comment:
I think this only needs either `-coverage ` or `-fprofile-arcs -ftest-coverage`

Reply:
I chose `-coverage`.